packages/platform-ios/src/commands/runIOS/index.ts

```diff
@@ -618,7 +618,7 @@ export default {
       description:
         'Path relative to project root where the Xcode project ' +
         '(.xcodeproj) lives.',
-      default: 'ios',
+      default: (ctx: Config) => ctx.project.ios ? ctx.project.ios.sourceDir : 'ios',
```

Comment:
`iOS` might be null when there's no `iOS` project but users run `react-native --help` and want to see all the commands. We could've used an empty string as well, but I guess sticking to `iOS` is also fine. 

No preference.